hack/builder/titus-executor-builder.sh

```diff
@@ -17,7 +17,7 @@ mv mount/titus-mount-block-device build/bin/linux-amd64/
 
 # tini
 (
-    mkdir -p build/tini
+    rm -rf build/tini && mkdir -p build/tini
```

Comment:
wont this fail if the dir `build` doesn't exist?

Reply:
No, `-f` will mean it wont fail